spec/fstab_spec.rb

```diff
@@ -1,18 +1,14 @@
 require 'stringio'
 
 describe LinuxAdmin::FSTab do
-  before do
```

Comment:
@carbonin I think this before should be changed to `subject { described_class.dup }`.  Than you can just change `LinuxAdmin::FSTab` to `subject` and the warnings should be gone.